### ISSUE TITLE
Emacs syntax highlighting mode (feature request)

### ISSUE THREAD

**Issue by g4v4g4i** [christianvoigt/argdown#91]
Would be great to support some other standard editors like the mighty emacs. 
In emacs syntax-highlighting for new languages can easily implemented (here's a howto: [http://ergoemacs.org/emacs/elisp_syntax_coloring.html]). Given it's huge user-base this could be interesting.

**Comment by christianvoigt** [comment]
I agree, emacs support would be great. With https://github.com/emacs-lsp/lsp-mode and the `@argdown/language-server` node package it would also be easy to get the linter and other code assistance running in emacs.

However, I currently do not have time for this and also think that this should be done by someone who actually uses emacs (I would help with integrating @argdown/language-server). I am open for pull requests for this (if this should be part of the official Argdown repo). 

Here are the links to the current syntax highlighting implementations:

* [VS Code](https://github.com/christianvoigt/argdown/blob/master/packages/argdown-vscode/syntaxes/argdown.tmLanguage.json)
* [Prism](https://github.com/christianvoigt/argdown/blob/master/packages/argdown-prism/index.js)
* [Codemirror](https://github.com/christianvoigt/argdown/blob/master/packages/argdown-codemirror-mode/index.js)
* [Pandoc](https://github.com/xylomorph/argdown-pandoc-highlighting/blob/master/argdown.xml)

@xylomorph: aren't you using emacs for Argdown?

**Comment by aindilis** [comment]
I'm also interested in an Emacs mode for Argdown (I tried to get lsp-mode working with Go to figure out how to use LSP in Emacs in order to get a start on writing the Emacs Argdown mode, but was unsuccessful).

**Comment by aindilis** [comment]
I'm attempting to get this working again.  I've installed using nvm/npm the @argdown/language-server package, but don't know how to run it using npx.
